pkg/asset/installconfig/ovirt/config.go

```diff
@@ -2,10 +2,13 @@ package ovirt
 
 import (
 	"crypto/x509"
+	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 
+	ovirtsdk "github.com/ovirt/go-ovirt"
+
```

Comment:
Keep all imports that are not standard imports, kube imports, or openshift imports in the same section.
```suggestion
```